Create Blog “how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management”

bsoutham opened the pull request:
Automatically generated by Netlify CMS

choukden reviewed (changes requested):
Hi Blaine, Nisha

Thank you very much for your new blog post contribution to HPE Developer Community.

I have made a few minor suggestions for edits.

Dale Rensing will go through the editorial review in the coming day or so.
 Please stay tuned for additional editorial suggestions for edits.

A couple of tips to review the suggestions and acknowledge them in GitHub:

•	Navigate to **File Changed** area to visualize the comments from start to the end, or navigate to **Conversation** area to visualize the comments per reviewer.
•	As you incorporate the suggestions for edit in your blog post entry, please click on button **Resolve Conversation** in GitHub.

Note we recommend you continue doing edits of your  blog post entry in the CMS Workflow editor while the entry is in the **"In Review"** column. Moving it back to the **"Drafts"** column would delete the Pull Request in GitHub.

Thank you for taking our edits into account. Once you have completed making your edits, please send an email to [denis.choukroun@hpe.com](mailto:denis.choukroun@hpe.com) and [dale.rensing@hpe.com](mailto:dale.rensing@hpe.com)  indicating that your blog post is ready for its final review before its publication.

Best regards,
Denis

drensing commented on content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md:
```diff
@@ -0,0 +1,136 @@
+tags:
+  - hpe-greenlake, hpe-greenlake-for-compute-ops-management
+---
+The Compute Ops Management REST API uses the OAuth 2.0 HPE GreenLake authentication flow. A limited lifetime access token needs to be provided in the header of each REST API request as the authorization bearer. The access token is associated with a subject (person or service) and has all the same permissions and privileges as the subject. The access token expires after 120 minutes.
```
It would be helpful to add a little more of a benefit statement to this introduction and to not get quite so detailed so early in the post. You want to draw the reader in. Maybe it could be stated something like this:

Common identity frameworks and protocols use token-based strategies to secure access to applications and resources. OAuth 2.0 is one of the most popular, using access tokens and refresh tokens to allow an application to access resources hosted by other servers on behalf of a user. The Compute Ops Management REST API uses the OAuth 2.0 HPEGreenLake authentication flow, where a limited lifetime access token is provided in the header of each REST API request as the authorization bearer. The access token is associated with a subject (person or service) and retains all the same permissions and privileges as the subject. In this blog post, I will discuss the essential steps required to generate this access token.

(I'd put the part about the access token expiring a little lower somewhere in the copy)

drensing commented on content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md:
```diff
@@ -0,0 +1,136 @@
+---
+The Compute Ops Management REST API uses the OAuth 2.0 HPE GreenLake authentication flow. A limited lifetime access token needs to be provided in the header of each REST API request as the authorization bearer. The access token is associated with a subject (person or service) and has all the same permissions and privileges as the subject. The access token expires after 120 minutes.
+
+This blog will discuss the essential steps required to generate the limited lifetime access token.
```
If you take my suggestion, then this line won't be necessary.

drensing commented on content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md:
```diff
@@ -0,0 +1,136 @@
+
+To configure your API Client Credentials, perform the following steps:
+
+1. Click on the **Manage** link on the header on the GreenLake platform
```
Place "HPE" in front of "GreenLake platform"

drensing commented on content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md:
```diff
@@ -0,0 +1,136 @@
+
+![GreenLake API Call with POSTMAN](/img/glcp_postman.png "GreenLake API Call with POSTMAN")
+
+# Deleting Client Credentials
```
"Deleting client credentials"

drensing reviewed (commented):
Hello, Nisha. Thank you for your excellent addition to the HPE Developer blog. I look forward to seeing this one published soon. Please have a look at my suggestions and see if they're helpful for you.

All the best,
Dale

bsoutham commented:
Thank you for the feedback.  I believe I've addressed all of the comments from both of you.
Preview can be found: 
https://deploy-preview-1478--hpe-dev-portal.netlify.app/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management/

bsoutham commented on content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md:
```diff
@@ -0,0 +1,136 @@
+
+## HPE GreenLake steps to obtain the access token for Compute Ops Management
+
+The user starts the process by logging in and authenticating into HPE GreenLake, which is authenticated by the Identity Provider (validated through username, password, Single Sign-On, or Multi-Factor Authentication). 
```
Changed - UI refers to it just as "HPE GreenLake"
